pkg/operator/k8sutil/pod.go

```diff
@@ -153,6 +153,21 @@ func MakeRookImage(version string) string {
 	return version
 }
 
+func PodsRunningWithLabel(clientset kubernetes.Interface, namespace, label string) (int, error) {
+	pods, err := clientset.CoreV1().Pods(namespace).List(metav1.ListOptions{LabelSelector: label})
+	if err != nil {
+		return 0, err
+	}
+
+	running := 0
+	for _, pod := range pods.Items {
+		if pod.Status.Phase == v1.PodPhase("Running") {
```

Comment:
is there not a const already defined in the k8s packages for this?

Reply:
yes, will update. Not sure how i missed that before.